src/components/ReleaseNotesVersions/ReleaseNotesVersions.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from "react";
+import { useHistory } from "@docusaurus/router";
+import Admonition from "@theme/Admonition";
+import { useVersions } from "@docusaurus/plugin-content-docs/client";
+import styles from "./ReleaseNotesVersions.module.scss";
+import ArchivedVersions from "../../../archiveVersions.json";
+import Select, { components, OptionProps } from "react-select";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Importing these on-page was causing jank on reloads in the nav cards, so that's why I ended up downloading the SVG and putting it in `/assets`.  Not sure if this might be the case here https://github.com/FortAwesome/react-fontawesome/issues/134

Reply:
I was worried about that too, but because it's a drop-down, the user never sees the reload during a refresh action.